Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,551 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
+
+public class LazyTensor : _AnyTensorHandle {
+    enum Handle {
+        /// Bool indicates if this concrete TFETensorhandle was a result of
+        /// materialization.
+       case conc(TFETensorHandle, Bool)
+        /// Bool indicates whether this is held by a swift variable,
+        /// i.e., is this a live tensor?
+       case sym(LazyTensorOperation, Int, Bool)
```

Comment:
```suggestion
       case symbolic(LazyTensorOperation, Int, Bool)
```

Other than the case name, the `Int` and the `Bool` associated values should be labeled for clarity. This is recommended by the "compensate for weak information" rule in the

Reply:
Done.